### ISSUE TITLE
Getting 404 not found when adding storybook-addon-design in .storybook/main.js 

### ISSUE THREAD

**Issue by hedaukartik** [pocka/storybook-addon-designs#114]
### Affected design types

- [X] figma
- [ ] experimental-figspec
- [X] link
- [ ] image
- [ ] iframe

### Describe the bug

When adding storybook-addon-design in .storybook/main.js storybook is not starting and getting 404 not found.

addons: [
'@storybook/addon-links',
'@storybook/addon-essentials',
'@storybook/preset-create-react-app',
'storybook-addon-design'
]

### How to reproduce the bug?

just by adding storybook-addon-design in main.js and restarting storybook getting 404 not found

### Expected behaviour

needed design tab in storybook

### Environment

- [X] Chrome / Chromium / Chromium based browsers
- [ ] Firefox
- [ ] macOS Safari
- [ ] iOS Safari
- [ ] Samsung Internet
- [ ] Other

### Affected versions

6.0.1

### Storybook versions

6.3.2

**Comment by pocka** [comment]
@hedaukartik Any console errors?

**Comment by jelling** [comment]
Same issue here using the latest version of the add-on and React 17. My project works fine both on its own and with Storybook so it's definitely the add-on.

Here are the console errors:

```
vendors~main.manager.bundle.js:108124 Uncaught TypeError: init is not a function
    at mountLazyComponent (vendors~main.manager.bundle.js:108124)
    at beginWork (vendors~main.manager.bundle.js:109517)
    at HTMLUnknownElement.callCallback (vendors~main.manager.bundle.js:94407)
    at Object.invokeGuardedCallbackDev (vendors~main.manager.bundle.js:94456)
    at invokeGuardedCallback (vendors~main.manager.bundle.js:94518)
    at beginWork$1 (vendors~main.manager.bundle.js:114421)
    at performUnitOfWork (vendors~main.manager.bundle.js:113233)
    at workLoopSync (vendors~main.manager.bundle.js:113164)
    at renderRootSync (vendors~main.manager.bundle.js:113127)
    at performSyncWorkOnRoot (vendors~main.manager.bundle.js:112750)

vendors~main.manager.bundle.js:110547 The above error occurred in one of your React components:
    at Lazy
    at Suspense
    at Scroller (http://localhost:6006/vendors~main.manager.bundle.js:21150:25)
    at render
    at nav
    at render
    at http://localhost:6006/vendors~main.manager.bundle.js:36655:29
    at ManagerConsumer (http://localhost:6006/vendors~main.manager.bundle.js:16062:28)
    at div
    at render
    at Sidebar (http://localhost:6006/vendors~main.manager.bundle.js:28148:28)
    at Layout (http://localhost:6006/vendors~main.manager.bundle.js:28320:5)
    at render
    at http://localhost:6006/vendors~main.manager.bundle.js:28647:20
    at div
    at render
    at http://localhost:6006/vendors~main.manager.bundle.js:40650:23
    at ReferenceWrapper (http://localhost:6006/vendors~main.manager.bundle.js:119275:5)
    at SizeMeRenderer (http://localhost:6006/vendors~main.manager.bundle.js:119328:29)
    at SizeAwareComponent (http://localhost:6006/vendors~main.manager.bundle.js:119405:9)
    at ThemeProvider
    at http://localhost:6006/vendors~main.manager.bundle.js:40816:25
    at ManagerConsumer (http://localhost:6006/vendors~main.manager.bundle.js:16062:28)
    at EffectOnMount (http://localhost:6006/vendors~main.manager.bundle.js:16051:24)
    at ManagerProvider (http://localhost:6006/vendors~main.manager.bundle.js:15910:5)
    at Location (http://localhost:6006/vendors~main.manager.bundle.js:3146:23)
    at QueryLocation (http://localhost:6006/vendors~main.manager.bundle.js:41650:24)
    at LocationProvider (http://localhost:6006/vendors~main.manager.bundle.js:3166:5)
    at r (http://localhost:6006/vendors~main.manager.bundle.js:27557:5365)
    at Root (http://localhost:6006/vendors~main.manager.bundle.js:40798:23)
    
 vendors~main.manager.bundle.js:108124 Uncaught TypeError: init is not a function
    at mountLazyComponent (vendors~main.manager.bundle.js:108124)
    at beginWork (vendors~main.manager.bundle.js:109517)
    at HTMLUnknownElement.callCallback (vendors~main.manager.bundle.js:94407)
    at Object.invokeGuardedCallbackDev (vendors~main.manager.bundle.js:94456)
    at invokeGuardedCallback (vendors~main.manager.bundle.js:94518)
    at beginWork$1 (vendors~main.manager.bundle.js:114421)
    at performUnitOfWork (vendors~main.manager.bundle.js:113233)
    at workLoopSync (vendors~main.manager.bundle.js:113164)
    at renderRootSync (vendors~main.manager.bundle.js:113127)
    at performSyncWorkOnRoot (vendors~main.manager.bundle.js:112750)

vendors~main.manager.bundle.js:110547 The above error occurred in the <LocationProvider> component:

  at LocationProvider (http://localhost:6006/vendors~main.manager.bundle.js:3166:5)
  at r (http://localhost:6006/vendors~main.manager.bundle.js:27557:5365)
  at Root (http://localhost:6006/vendors~main.manager.bundle.js:40798:23)
    
vendors~main.manager.bundle.js:108124 Uncaught TypeError: init is not a function
  at mountLazyComponent (vendors~main.manager.bundle.js:108124)
  at beginWork (vendors~main.manager.bundle.js:109517)
  at HTMLUnknownElement.callCallback (vendors~main.manager.bundle.js:94407)
  at Object.invokeGuardedCallbackDev (vendors~main.manager.bundle.js:94456)
  at invokeGuardedCallback (vendors~main.manager.bundle.js:94518)
  at beginWork$1 (vendors~main.manager.bundle.js:114421)
  at performUnitOfWork (vendors~main.manager.bundle.js:113233)
  at workLoopSync (vendors~main.manager.bundle.js:113164)
  at renderRootSync (vendors~main.manager.bundle.js:113127)
  at performSyncWorkOnRoot (vendors~main.manager.bundle.js:112750)

POST http://localhost:6006/runtime-error 404 (nor found) 

```

**Comment by pocka** [comment]
https://github.com/storybookjs/storybook/issues/13670

Seems like this.

**Comment by pocka** [comment]
I've added `react` and `react-dom` to `peerDependencies` with `"peerDependenciesMeta": { "...": { "optional": true } }}`. The patch is available from v7.0.0-alpha.0. If this issue still happens regardless of the patch and there is something this addon can do, please open a new issue.
